packages/@ama-sdk/generator-sdk/project.json

```diff
@@ -40,7 +40,9 @@
           "packages/@ama-sdk/generator-sdk/src/**/swagger-codegen-cli.jar",
           "packages/@ama-sdk/generator-sdk/src/**/config/*.json",
           "packages/@ama-sdk/generator-sdk/src/generators/**/target/*.jar",
-          "packages/@ama-sdk/generator-sdk/src/generators/*/templates/**/*",
+          "packages/@ama-sdk/generator-sdk/src/generators/shell/templates/**/*",
```

Comment:
Why this change, the `*` was not doing the job?

Reply:
The `generators/*/templates` allowed in the packaging all the files inside `templates` folder for all the generators, not only the templates files for schematics generator. 
This was a problem for the packaging of java part of generators, because along with the `.jar` files in `target` folder we got also the `.class` files, which break the generators at runtime because it is trying to execute the them.
Ex: `generator-sdk/src/generators/java-client-core/templates/target` has matched with the `/generators/*/templates/**/*`, and that's why I specified exactly which are the `schematics` tempaltes to package.